client/rest-high-level/src/main/java/org/elasticsearch/client/ml/GetDataFrameAnalyticsRequest.java

```diff
@@ -33,10 +33,12 @@
 public class GetDataFrameAnalyticsRequest implements Validatable {
 
     public static final ParseField ALLOW_NO_MATCH = new ParseField("allow_no_match");
```

Comment:
BTW, is there any reason for `ALLOW_NO_MATCH` to be `ParseField` rather than `String`?

Reply:
probably not, but just didn't touch the existing field :)